iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/IoTDBConnector.java

```diff
@@ -53,6 +61,11 @@ public abstract class IoTDBConnector implements PipeConnector {
 
   protected boolean isTabletBatchModeEnabled = true;
 
+  protected final AtomicLong commitIdGenerator = new AtomicLong(0);
```

Comment:
To ensure the ordering of the commiting across connectors, it may be better to make the 3 variables static, to let all the connectors share one commitId

Reply:
how about connectors with same attribute string share one commit id?

Reply:
This will not affect the correctness, for the global commit order ensures the local commit order of each pipe sharing the same connector.

> how about connectors with same attribute string share one commit id?